exept/exeptions.py

```diff
@@ -83,4 +83,19 @@ class UserAlreadyExistError(CustomQSportException):
     def __init__(self, date_value):
         self.date_value = date_value
         message = f"User with such data already exist'"
+        super().__init__(message)
+
+class IncorrectLogInStrategyMethod(CustomQSportException):
+    status_code = 404
+
+    def __init__(self, data_value):
+        message = f"{data_value} is not a log in method in this app"
+        super().__init__(message)
+
+
+class InvalidDataForGoogleLogIn(CustomQSportException):
```

Comment:
Imagine we have Facebook, GitHub, Twitter (X) and meny other Auth providers. If we were to create an exception for each one of these how would they differ? Same status code, message differs only by auth_provider's name. So it's probably better to create an exception that is a bit more generic, like:

`InvalidAuthenticationDataError`

which would have: `message = f'{auth_provider_name} data is invalid'`